src/content-script/styles.css

```diff
@@ -74,3 +74,7 @@
   white-space: pre-wrap;
   word-break: break-all;
 }
+
+.hljs {
+  padding: 0 !important;
+}
```

Comment:
Don't do this, this might affect other elements using this class in the page. Put it inside `.chat-gpt-container`

Reply:
![image](https://user-images.githubusercontent.com/41566813/207571408-3f80e6c7-45c3-42d1-abff-d29923907a18.png)
Does this looks better?

Reply:
that's right